src/test/java/com/adyen/LegalEntityManagementTest.java

```diff
@@ -345,4 +346,23 @@ public void TestNewFieldsInResponseDoesNotThrowError() throws IOException {
             Assert.fail();
         }
     }
+
+    @Test
+    public void TestOverrideField() throws IOException, ApiException, HTTPClientException {
+        Client client = createMockClientFromFile("mocks/management/list-merchants.json");
```

Comment:
should it be this response, or something similar?
```suggestion
        Client client = createMockClientFromFile("mocks/legalentitymanagement/OnboardingLink.json");
```